library/psa_crypto.c

```diff
@@ -1011,6 +1013,16 @@ psa_status_t psa_wipe_key_slot( psa_key_slot_t *slot )
      * been wiped. Clear remaining metadata. We can call memset and not
      * zeroize because the metadata is not particularly sensitive. */
     memset( slot, 0, sizeof( *slot ) );
+
+    do
+    {
+        rnd_status = psa_generate_random( (uint8_t *) &slot->key_id,
+                                          sizeof(slot->key_id) );
```

Comment:
What if the slot is reused immediately afterwards?

Reply:
I'm not sure if I follow what's the problem here - if you mean immediately as in after calling this function - we should have a mutex in place in the future. If you mean after the call to slot wipe - no problem here.